docs/agent-guard-secrets-python-get-started.md

```diff
@@ -0,0 +1,190 @@
+# Getting started with the credentials module
+
+## Add the dependencies
+
+This module is available on PyPI. Add agent-guard-core to your project:
+
+### uv
+```bash
+uv add agent-guard-core
+```
+### pip
+```bash
+pip3 install agent-guard-core
+```
+
+**_NOTE:_** Please ensure you are using Poetry version >=2.1.1.
```

Comment:
Why is Poetry relevant if we give examples of pip and uv?

Reply:
@rafis3  I didn't touch this page.. this was from the original Get_started page (see main)